File: 2529-maximum-count-of-positive-integer-and-negative-integer/2529-maximum-count-of-positive-integer-and-negative-integer.cpp
class Solution {
public:
    int maximumCount(vector<int>& nums) {
        int n,p=0;
        for(int c:nums){
            if(c>0){
                p++;
            }
            else if(c<0){
                n++;
            }
        }
        return (n>p)?n:p;
    }
};

File: 2529-maximum-count-of-positive-integer-and-negative-integer/2529-maximum-count-of-positive-integer-and-negative-integer.py
from typing import List

class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        nos, pos = 0, 0
        
        for num in nums: 
            if num < 0:
                nos += 1
            elif num > 0:  
                pos += 1
        
        return max(nos, pos) 
